src/docs/sphinx/quickstart.rst

```diff
@@ -325,35 +329,70 @@ Note that the version format output by the above command is not the same as that
 If you are not using an M2 or M3 Mac, you will need to change the ``target`` for the compiler to ``x86_64`` or ``aarch64`` for Intel and M1-based Macs, respectively.
 Similarly, you need to set the ``operating_system`` to the proper value if you are not using ``sonoma`` (MacOS 14.X).
 
-If you want to install the devtools, you should also add the following under ``packages`` in the ``spack.yaml`` files.
+Installing Python Developer Tools
```

Comment:
"Python Developer Tools" to me means that we are developing Python, which we aren't. Maybe change this to just "Project Developer Tools"